flexible-classroom/reference/classroom-api.mdx

```diff
@@ -45,11 +45,11 @@ Call this method to create a classroom. After it is created, the classroom is re
 
 **URL parameters**
 
-Pass the following parameters in the URL.
+Pass the following parameters in the URL:
 
 | Parameter  | Type   | Description                                                  |
 | :--------- | :----- | :----------------------------------------------------------- |
-| `region`   | String | (Required) The region for connection. Flexible Classroom supports the following regions:<li>`cn`: Mainland China.</li><li>`ap`: Asia Pacific.</li><li>`eu`: Europe.</li><li>`na`: North America.</li> |
+| `region`   | String | (Required) The region for connection. Flexible Classroom supports the following regions:<ul><li>`cn`: Mainland China.</li><li>`ap`: Asia Pacific.</li><li>`eu`: Europe.</li><li>`na`: North America.</li></ul> |
```

Comment:
These `<ul>` tags are to fix ugly looking bullet points like these:
![image](https://user-images.githubusercontent.com/42718091/212846450-db8b05ac-0d5b-40f6-835f-a8a3ffec0d56.png)